homeassistant/components/renson/switch.py

```diff
@@ -0,0 +1,91 @@
+"""Breeze switch of the Renson ventilation unit."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from renson_endura_delta.field_enum import CURRENT_LEVEL_FIELD, DataType
+from renson_endura_delta.renson import Level, RensonVentilation
+
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import RensonCoordinator
+from .const import DOMAIN
+from .entity import RensonEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RensonBreezeSwitch(RensonEntity, SwitchEntity):
+    """Provide the breeze switch."""
+
+    _attr_icon = "mdi:weather-dust"
+    _attr_name = "Breeze"
+
+    def __init__(
+        self,
+        api: RensonVentilation,
+        coordinator: RensonCoordinator,
+    ) -> None:
+        """Initialize class."""
+        super().__init__("breeze", api, coordinator)
+
+        self._state = None
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if switch is on."""
+        if self._state is None:
+            return False
+
+        return self._state
```

Comment:
Please use `_attr_is_on` instead of `_state`, that way you can remove this property